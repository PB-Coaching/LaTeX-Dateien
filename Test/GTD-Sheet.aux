\relax 
\bibstyle{address}
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\citation{c.geuzaine}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}General description}{2}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Package inclusion and options}{2}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Making a citation}{2}{subsection.2.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}The \texttt  {bst} styles}{2}{subsection.2.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.4}The \texttt  {bib} file fields}{3}{subsection.2.4}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Customization}{5}{section.3}}
\newlabel{sec:custom}{{3}{5}{Customization\relax }{section.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Dimensions}{5}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}Flags and formats}{5}{subsection.3.2}}
\newlabel{sec:flagsformats}{{3.2}{5}{Flags and formats\relax }{subsection.3.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Using both \texttt  {directory} and \texttt  {bibliography}}{6}{section.4}}
\newlabel{sec:usingboth}{{4}{6}{Using both \texttt {directory} and \texttt {bibliography}\relax }{section.4}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Generating directories with hypertext links}{6}{section.5}}
\@writefile{toc}{\contentsline {section}{\numberline {6}Generating HTML, vCard or LDIF directories}{6}{section.6}}
\@writefile{toc}{\contentsline {section}{\numberline {7}Example}{7}{section.7}}
\newlabel{sec:example}{{7}{7}{Example\relax }{section.7}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1}Source file}{8}{subsection.7.1}}
\citation{*}
\bibdata{family,business}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2}Output}{9}{subsection.7.2}}
\newlabel{sec:output}{{7.2}{9}{Output\relax }{subsection.7.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {8}Contributors}{9}{section.8}}
\@writefile{toc}{\contentsline {section}{\numberline {9}Versions}{9}{section.9}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Summary of customization commands}}{11}{table.1}}
\newlabel{tab:commands}{{1}{11}{Summary of customization commands\label {tab:commands}\relax }{table.1}{}}
