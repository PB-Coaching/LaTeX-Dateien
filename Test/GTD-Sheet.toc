\contentsline {section}{\numberline {1}Introduction}{1}{section.1}
\contentsline {section}{\numberline {2}General description}{2}{section.2}
\contentsline {subsection}{\numberline {2.1}Package inclusion and options}{2}{subsection.2.1}
\contentsline {subsection}{\numberline {2.2}Making a citation}{2}{subsection.2.2}
\contentsline {subsection}{\numberline {2.3}The \texttt {bst} styles}{2}{subsection.2.3}
\contentsline {subsection}{\numberline {2.4}The \texttt {bib} file fields}{3}{subsection.2.4}
\contentsline {section}{\numberline {3}Customization}{5}{section.3}
\contentsline {subsection}{\numberline {3.1}Dimensions}{5}{subsection.3.1}
\contentsline {subsection}{\numberline {3.2}Flags and formats}{5}{subsection.3.2}
\contentsline {section}{\numberline {4}Using both \texttt {directory} and \texttt {bibliography}}{6}{section.4}
\contentsline {section}{\numberline {5}Generating directories with hypertext links}{6}{section.5}
\contentsline {section}{\numberline {6}Generating HTML, vCard or LDIF directories}{6}{section.6}
\contentsline {section}{\numberline {7}Example}{7}{section.7}
\contentsline {subsection}{\numberline {7.1}Source file}{8}{subsection.7.1}
\contentsline {subsection}{\numberline {7.2}Output}{9}{subsection.7.2}
\contentsline {section}{\numberline {8}Contributors}{9}{section.8}
\contentsline {section}{\numberline {9}Versions}{9}{section.9}
