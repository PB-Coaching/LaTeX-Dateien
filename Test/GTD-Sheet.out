\BOOKMARK [1][-]{section.1}{Introduction}{}
\BOOKMARK [1][-]{section.2}{General description}{}
\BOOKMARK [2][-]{subsection.2.1}{Package inclusion and options}{section.2}
\BOOKMARK [2][-]{subsection.2.2}{Making a citation}{section.2}
\BOOKMARK [2][-]{subsection.2.3}{The bst styles}{section.2}
\BOOKMARK [2][-]{subsection.2.4}{The bib file fields}{section.2}
\BOOKMARK [1][-]{section.3}{Customization}{}
\BOOKMARK [2][-]{subsection.3.1}{Dimensions}{section.3}
\BOOKMARK [2][-]{subsection.3.2}{Flags and formats}{section.3}
\BOOKMARK [1][-]{section.4}{Using both directory and bibliography}{}
\BOOKMARK [1][-]{section.5}{Generating directories with hypertext links}{}
\BOOKMARK [1][-]{section.6}{Generating HTML, vCard or LDIF directories}{}
\BOOKMARK [1][-]{section.7}{Example}{}
\BOOKMARK [2][-]{subsection.7.1}{Source file}{section.7}
\BOOKMARK [2][-]{subsection.7.2}{Output}{section.7}
\BOOKMARK [1][-]{section.8}{Contributors}{}
\BOOKMARK [1][-]{section.9}{Versions}{}
